lib/msf/core/exploit/remote/auto_check.rb

```diff
@@ -36,10 +36,10 @@ def with_prepended_auto_check
       return yield
     end
 
-    print_status('Executing automatic check (disable AutoCheck to override)')
+    print_status('Running automatic check ("set AutoCheck false" to disable)')
```

Comment:
This is more consistent with the option description and behavior (and avoids semantic satiation). Also adds a handy example.